ocaml/database/block_device_io.ml

```diff
@@ -325,7 +325,7 @@ let transfer_data_from_sock_to_fd sock dest_fd available_space target_response_t
                (* Check that there's enough space *)
                if available_space - !total_length < len then raise NotEnoughSpace;
                (* Otherwise write it *)
-               Unixext.time_limited_write dest_fd len chunk target_response_time;
+               Unixext.time_limited_write dest_fd len (Bytes.unsafe_of_string chunk) target_response_time;
```

Comment:
Before merging this PR, I plan to modify `Unixext.read_data_in_chunks` so that the `chunk` is bytes. This would make the signature closer to the `Unix` functions and reduce the amount of conversions that we need.